tests/test_scripts.py

```diff
@@ -1,22 +1,31 @@
 import os
 from entity_resolver.scripts import run
+import json
 
+# class TestArxiv:
+#
+#     def test_dummy(self):
+#         file_path = 'file.txt'
+#         run(['norm-arxiv', '', file_path])
+#         with open(file_path, 'r') as f:
+#             assert f.read() == 'arxiv'
+#         os.remove(file_path)
 
-class TestArxiv:
 
-    def test_dummy(self):
-        file_path = 'file.txt'
-        run(['norm-arxiv', '', file_path])
-        with open(file_path, 'r') as f:
-            assert f.read() == 'arxiv'
-        os.remove(file_path)
```

Comment:
Although both data files seem to share the same function, I would still suggest writing two tests for the two processing functions separately, and not removing the `TestArxiv` class and its testing function.